src/Command/BhaktasCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace App\Command;
+
+use Cake\Console\Arguments;
+use Cake\Console\Command;
+use Cake\Console\ConsoleIo;
+use Cake\I18n\Time;
+use Cake\I18n\Date;
+use Cake\Mailer\Email;
+
+class BhaktasCommand extends Command
+{
+	
+	Const ONKENTES_TAMOGATO = 2;
+	/**
+     * Initialize method
+     *
+     * @param array $config The configuration for the Table.
+     * @return void
+     */
+	public function initialize()
+    {
+        parent::initialize();
+        $this->loadModel('Bhaktas');
+    }
+    public function execute(Arguments $args, ConsoleIo $io)
+    {
+		$years = $io->ask('Insert the Years');
```

Comment:
This is only good if we run manually from the command line. With cron we need t pass arguments. So you have to use `getOptionParser()` and than `$this->params`
I am sorry it seems I did not expressed clearly what we need here